pkg/ccl/streamingccl/settings.go

```diff
@@ -119,3 +119,12 @@ var DumpFrontierEntries = settings.RegisterDurationSetting(
 	0,
 	settings.NonNegativeDuration,
 )
+
+// ReplicateSpanConfigsEnabled controls whether we replicate span
+// configurations from the source system tenant to the destination system
+// tenant.
+var ReplicateSpanConfigsEnabled = settings.RegisterBoolSetting(settings.SystemOnly,
+	"physical_replication.span_configs.enabled",
```

Comment:
[consistency nits]: settings.SystemOnly on the next line like the rest, `physical_replication.consumer.span_configs.enabled`